src/logics/morphoblue/configs.ts

```diff
@@ -19,6 +19,34 @@ export interface Config {
 }
 
 export const configs: Config[] = [
+  {
+    chainId: common.ChainId.mainnet,
+    contract: {
+      Morpho: '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb',
+      MorphoFlashLoanCallback: '0x24D5b6b712D1f0D0B628E21E39dBaDde3f28C56e',
+    },
+    // get meta morpho addresses from MetaMorphoFactory CreateMetaMorpho events
+    // get market ids from meta morpho supply/withdraw queues
+    // zero oracle address is excluded
+    markets: [
+      {
+        id: '0xc54d7acf14de29e0e5527cabd7a576506870346a78a11a6762e2cca66322ec41',
```

Comment:
Did you consider to handle the market id like [Compound v3](https://github.com/dinngo/protocolink-logics/blob/fabecd1829a7f9437e1a8e7d93148c1439cf8c11/src/logics/compound-v3/configs.ts#L5)? Might be a little different on naming but will be easier to understand i think.

Reply:
I've thought about that but the Morpho markets would be more complicated (say WETH-USDC, any of the factors [oracle, irm, lltv] changed becomes a different market). It would be easier to use the id directly but still open for any suggestions.